pkg/skaffold/debug/transform.go

```diff
@@ -14,6 +14,39 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/*
+The `debug` package transforms Kubernetes pod-bearing resources so as to configure containers
+for remote debugging as suited for a container's runtime technology.  This package defines
+a _container transformer_ interface. Each transformer implementation should do the following:
+
+1. The transformer should modify the container's entrypoint, command arguments, and environment to enable debugging for the appropriate language runtime.
+2. The transformer should expose the port(s) required to connect remote debuggers.
+3. The transformer should identify any additional support files required to enable debugging (e.g., the `Delve` debugger for Go).
```

Comment:
presumably this `go/Delve` reference shouldn't be included in the docs until go is actually supported?

Reply:
Foreshadowing!